src/main/java/ch/njol/skript/sections/SecConditional.java

```diff
@@ -118,10 +118,11 @@ public boolean init(Expression<?>[] exprs,
 		multiline = parseResult.regexes.size() == 0 && type != ConditionalType.ELSE;
 
 		// ensure this conditional is chained correctly (e.g. an else must have an if)
-		SecConditional lastIf;
+		SecConditional precedingIf;
 		if (type != ConditionalType.IF) {
-			lastIf = getClosestIf(triggerItems);
-			if (lastIf == null) {
+			// find the latest 'if' section so that we can ensure this section is placed properly (e.g. ensure a 'if' occurs before an 'else')
+			precedingIf = getPrecedingConditional(triggerItems, ConditionalType.IF);
```

Comment:
this isn't needed if `type == ConditionalType.THEN` right? maybe you could do something like:
```java
if (type == ConditionalType.THEN) {
  precedingIf = getPrecedingConditional(triggerItems, null);
  // the rest of the stuff
} else { // ELSE/ELSE IF
  precedingIf = getPrecedingConditional(triggerItems, ConditionalType.IF);
  // the rest of the stuff
}
```
since both start with setting precedingIf, you could also just select the ConditionalType from a ternary `type == ConditionalType.THEN ? null : ConditionalType.IF`